docs/development_suite/set-up-infrastructure/configure-provider.mdx

```diff
@@ -1,258 +1,163 @@
-
-For `Vault` provider, the currently supported credential types are **token** and **m2m**. The object will be composed as follows:
-- **credentials (object)** (required):
-  - **type (string)** (required): "token"
-  - **content (object)** (required):
-    - **accessToken (string)** (required)
-
-* **credentials (object)**(required):
-  * **type (string)** (required): "m2m"
-  * **content (object)** (required):
-    * **token (string)** (required)
-    * **accessTokenURL (string)** (required)
-
-For `Jenkins` provider, the currently supported credential types are **token** and **OAuth2** (Client Credentials). The object will be composed as follows:
-- **credentials (object)** (required):
-  - **type (string)** (required): "token"
-  - **content (object)** (required):
-    - **accessToken (string)** (required)
-
-* **credentials (object)**(required):
-  * **type (string)** (required): "client_credentials"
-  * **content (object)** (required):
-    * **clientId (string)** (required)
-    * **clientSecret (string)** (required)
-    * **accessTokenURL (string)** (required)
+In this step, you are required to choose the specific provider you want to use among the ones supported by the Console and, for the chosen provider, the capabilities you want to use it for ("Git Provider", "Secret Manager", "CI/CD Tool").
+In case the provider offers only one capability, clicking on it will go directly to step 2 and the capability will be selected by default.
 
+### Step 2: Provider Details
+
+![Add provider 2](img/add-provider-2.png)
+
+In this step, you can insert some general details about your provider:
+
+* **Provider name** (*required*): a human-readable name for the provider;
+* **API Base URL** (*required*): the reference endpoint URL for calls to the provider APIs;
+* **Base URL** (*required*): the reference endpoint URL for the provider;
+* **Description**: a brief description of the provider.
+
+### Step 3: Credentials
+
+![Add provider 3](img/add-provider-3.png)
+
+In this step, you are required to insert information about the authentication method to the provider, so that the Console has the necessary permissions to perform all read/write operations on it:
+
+* **Credentials type** (*required*): the type of credentials used to authenticate requests to the provider. It can be one of "Token," "M2M," and "OAuth2".
+
+Depending on the credentials type, the other information to be entered will be:
+
+* For credentials of the type "Token":
+  - **Access token** (*required*): token that allows the Console to authenticate against the APIs of the provider (temporary credentials).
+* For credentials of the type "M2M":
+  - **Token** (*required*): token that allows the Console to authenticate against the APIs of the provider (long-lived credentials);
+  - **Access token URL** (*required*): the reference authorization endpoint URL that, to requests that include the authentication token, responds by generating temporary credentials.
```

Comment:
```suggestion
Depending on the credentials type, the other information to be entered will be:

* For credentials of the type "Token":
  - **Access token** (*required*): a token that allows the Console to authenticate against the APIs of the provider (temporary credentials).
* For credentials of the type "M2M":
  - **Token** (*required*): a token that allows the Console to authenticate against the APIs of the provider (long-lived credentials);
  - **Access token URL** (*required*): the reference authorization endpoint URL that, to requests that include the authentication token, responds by generating temporary credentials.
```

Reply:
Secondo me "a" è sbagliato, perché non è UN token ma IL token specifico che permette determinate azioni. Posso mettere "the" ma secondo me suona meglio senza articolo